### PR TITLE
TELCODOCS-1154 - release notes for RAN 4.13 new features and bugs

### PR DESCRIPTION
**Pull request by aireilly**
Release notes for RAN 4.13 new features and bugs.  

Version(s):
enterprise-4.13

Issue:
https://issues.redhat.com/browse/TELCODOCS-1154

Link to docs preview:
* [Tech Preview status](https://58179--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-scalability-tech-preview)
* [New features](https://58179--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-scalability-and-performance)
* [Known issues](https://58179--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-ran-known-issues)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc
Comment (amolnar-rh) [inline release_notes/ocp-4-13-release-notes.adoc:320]: RN for [OCPVE-226](https://issues.redhat.com/browse/OCPVE-226)

@suleymanakbas91 Please take a look when you have the time.

File: release_notes/ocp-4-13-release-notes.adoc
Comment (suleymanakbas91) [reply release_notes/ocp-4-13-release-notes.adoc]: This one looks good.

File: release_notes/ocp-4-13-release-notes.adoc
Comment (suleymanakbas91) [reply release_notes/ocp-4-13-release-notes.adoc]: `you can apply labels to LVM {lvms} resources to enable workload partitioning.` The necessary labels are already applied to lvms resources so we could change this to something like `the required labels are applied to lvms resources so you can enable workload partitioning`.

Comment (suleymanakbas91) [reply release_notes/ocp-4-13-release-notes.adoc]: Sounds good :)

File: release_notes/ocp-4-13-release-notes.adoc
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:671]: @aneeshkp @josephdrichard Can you LGTM on this please.

File: release_notes/ocp-4-13-release-notes.adoc
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:680]: @aneeshkp @josephdrichard Can you LGTM on this please.

File: release_notes/ocp-4-13-release-notes.adoc
Comment (kcarmichael08) [inline release_notes/ocp-4-13-release-notes.adoc:2011]: suggestion: "right" = "immediately" ?

**Review by kcarmichael08 — COMMENTED** [review]
Couple of comments; otherwise, looks good to me!

File: release_notes/ocp-4-13-release-notes.adoc
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:2015]: @Missxiaoguo Can you LGTM on this 4.13 known issue text please.

File: release_notes/ocp-4-13-release-notes.adoc
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:2018]: @snits Can you LGTM on this 4.13 known issue text please.

File: release_notes/ocp-4-13-release-notes.adoc
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:2021]: @pixelsoccupied can you Can you LGTM on this 4.13 known issue text please.

Comment (pixelsoccupied) [reply release_notes/ocp-4-13-release-notes.adoc:2021]: LGMT thanks!

File: release_notes/ocp-4-13-release-notes.adoc
Comment (aireilly) [reply release_notes/ocp-4-13-release-notes.adoc]: Updated, PTAL

File: release_notes/ocp-4-13-release-notes.adoc
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:2148]: @orosp can you review/LGTM on this 4.13 known issue please.

File: release_notes/ocp-4-13-release-notes.adoc
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:2148]: @orosp can you LGTM on this OCP 4.13 known issue please

Comment (aireilly) [reply release_notes/ocp-4-13-release-notes.adoc:2148]: LGTM from @orosp on slack

File: release_notes/ocp-4-13-release-notes.adoc
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:2169]: @jzding Can you review this 4.13 known issue please?

File: release_notes/ocp-4-13-release-notes.adoc
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:2180]: @jlelli can you please LGTM on this 4.13 known issue

File: release_notes/ocp-4-13-release-notes.adoc
Comment (jzding) [reply release_notes/ocp-4-13-release-notes.adoc:2169]: the `linuxptp-daemon` daemon pod intermittently fails to deploy.

**Review by jzding — APPROVED** [review]
/lgtm

File: release_notes/ocp-4-13-release-notes.adoc
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:2475]: @yliu127 can you LGTM on this please?

Comment (yliu127) [reply release_notes/ocp-4-13-release-notes.adoc:2475]: lgtm

File: release_notes/ocp-4-13-release-notes.adoc
Comment (abhatt-rh) [inline release_notes/ocp-4-13-release-notes.adoc:2510]: Per IBM Style:
> Do not start a sentence with a lowercase term; rewrite the sentence.

Further, the linked Jira issue points to `oslat` and not `cyclictest`. Based on [Measuring latency](https://docs.openshift.com/container-platform/4.12/scalability_and_performance/cnf-performing-platform-verification-latency-tests.html#cnf-measuring-latency_cnf-latency-tests), I understand that these are different tests. Moreover, the preceding point addresses the issue for `oslat`. Kindly verify.

File: release_notes/ocp-4-13-release-notes.adoc
Comment (abhatt-rh) [inline release_notes/ocp-4-13-release-notes.adoc:2507]: Per IBM Style:
> Do not start a sentence with a lowercase term; rewrite the sentence.

Consider the following:
```suggestion
* The `oslat` performance tests that run on a {sno} cluster running {product-title} 4.13 show a max latency result greater than 20 microseconds.
```

**Review by tliu2021 — COMMENTED** [review]
/lgtm

**Review by abhatt-rh — COMMENTED** [review]
Hi @aireilly 
Nice work on the RAN release notes!
I added a couple of comments for your consideration. It seemed as though some bits were not peer reviewed (I might be wrong). You may also need to update the preview for the _Known issues_ section because the last couple of updates in that section aren't showing up.

Also, this PR needs QE approval.

File: release_notes/ocp-4-13-release-notes.adoc
Comment (aireilly) [reply release_notes/ocp-4-13-release-notes.adoc:1052]: It was added by a member of the Core team.

**Comment by josclark42** [comment]
OCPBUGS-9272 note regarding disk partitions in SiteConfig LGTM

File: release_notes/ocp-4-13-release-notes.adoc
Comment (aireilly) [reply release_notes/ocp-4-13-release-notes.adoc:1432]: Not going to change this one :)

File: release_notes/ocp-4-13-release-notes.adoc
Comment (aireilly) [reply release_notes/ocp-4-13-release-notes.adoc:2510]: Updated

File: release_notes/ocp-4-13-release-notes.adoc
Comment (aireilly) [reply release_notes/ocp-4-13-release-notes.adoc]: Created https://github.com/redhat-documentation/vale-at-red-hat/pull/529 thanks

Comment (abhatt-rh) [reply release_notes/ocp-4-13-release-notes.adoc]: > Created [redhat-documentation/vale-at-red-hat#529](https://github.com/redhat-documentation/vale-at-red-hat/pull/529) thanks
 
Ooh, I like this approach. I could create so many of these from my peer review week :D
Thank you!

**Comment by linuxqiao** [comment]
/lgtm

Both RHELPLAN-155443 and RHELPLAN-155460 still in investigation.

**Comment by yuying1987** [comment]
lgtm for RHELPLAN-13823

**Pull request merged**
